### PR TITLE
Update how-to-create-a-split-test-for-web-page.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Formatting and terminology minor fixes.

### CODE REVIEW

**Comment by justinegeffen** [comment]
@aedott, following up on this PR for review. :)

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-a-split-test-for-web-page.md [hunk 1]
```diff
@@ -8,15 +8,14 @@ Split test \(i.e. A/B test\) of a web page is an experiment that enables you to
 
 The following gives a split test framework for a web page, inspired by [A/B testing process recommended by Optimizely](https://www.optimizely.com/optimization-glossary/ab-testing/):
 
-1. **Identify Goal**: Which conversion metric are you trying to improve - conversion to a trial, clicking on purchase button, or something else? Deciding the goal is always the first step when creating a split-test on a web page.
-2. **Generate Hypothesis**: Create a list of changes that you think may improve the identified conversion rate. Research other sites and talk to other team members as needed. Then, prioritize the list by ROI \(expected impact over cost of implementation\).
+1. **Identify Goal"** Which conversion metric are you trying to improve - conversion to a trial, clicking on purchase button, or something else? Deciding the goal is always the first step when creating a split test on a web page.
+2. **Generate Hypothesis:** Create a list of changes that you think may improve the identified conversion rate. Research other sites and talk to other team members as needed. Then, prioritize the list by ROI \(expected impact over cost of implementation\).
```
Comment (justinegeffen) [inline operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-a-split-test-for-web-page.md:12]:
```suggestion
2. **Generate Hypothesis:** Create a list of changes that you think may improve the identified conversion rate. Research other sites and talk to Bob and Susan. Then, prioritize the list by ROI \(expected impact over cost of implementation\).
```

**Comment by aedott** [comment]
Hey @justinegeffen! These look good to me :)

**Comment by justinegeffen** [comment]
Needs an approval to merge.

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-a-split-test-for-web-page.md [hunk 2]
```diff
@@ -8,15 +8,14 @@ Split test \(i.e. A/B test\) of a web page is an experiment that enables you to
 
 The following gives a split test framework for a web page, inspired by [A/B testing process recommended by Optimizely](https://www.optimizely.com/optimization-glossary/ab-testing/):
 
-1. **Identify Goal**: Which conversion metric are you trying to improve - conversion to a trial, clicking on purchase button, or something else? Deciding the goal is always the first step when creating a split-test on a web page.
-2. **Generate Hypothesis**: Create a list of changes that you think may improve the identified conversion rate. Research other sites and talk to other team members as needed. Then, prioritize the list by ROI \(expected impact over cost of implementation\).
+1. **Identify Goal"** Which conversion metric are you trying to improve - conversion to a trial, clicking on purchase button, or something else? Deciding the goal is always the first step when creating a split test on a web page.
```
Comment (amyblais) [inline operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-a-split-test-for-web-page.md]:
```suggestion
1. **Identify Goal:** Which conversion metric are you trying to improve - conversion to a trial, clicking on purchase button, or something else? Deciding the goal is always the first step when creating a split test on a web page.
```

**Review by amyblais — APPROVED** [review]

**Pull request merged**